### ISSUE TITLE
Request: Saving files in an open directory

### ISSUE THREAD

**Issue by meadows19** [msoultanidis/quillnote#23]
Hello. This is a beautiful app, and I thank you for making it.

I have one suggestion. It would be excellent if Quillnoteb saved its files in a user chosen accessible directory, rather than in an internal user inaccessible directory. 

That way, I can sync the folder and all the .md files inside it to my PC using Resilio Sync or Syncthing, and I can continue editing there. 

The problem as Quillnote is designed right now,  is that I have to export the files from Quillnote every time I want to sync them to my PC.

Another important reason for this approach is data portability,  so that we are not locked in to any specific app in the future. 

For an example of an app that works like this,  please see Noteless (especially in Dendron mode): 

https://github.com/redsolver/noteless

So in summary:

1) all notes should be preferably saved in one single directory (or subdirectories of the base directory). 
2) the user should choose the directory. 
3) all files should be in open formats such as txt or md

Please let me know what you think of this idea!

Thank you again for your work on this app.

**Comment by meadows19** [comment]
This is what I want to use Quillnote with on Windows:

https://notable.app/

It just uses a folder full of .md files. 

If Quillnote does the same,  I can edit on mobile or on my PC and the files will always stay in sync using Resilio Sync.

**Comment by meadows19** [comment]
Another android app that works this way is Markor:

https://github.com/gsantner/markor

**Comment by gergesh** [comment]
See #9

**Comment by msoultanidis** [comment]
Related to https://github.com/msoultanidis/quillnote/issues/9
